src/renderer/components/settings/settings.css

```diff
@@ -11,4 +11,62 @@
 
 .vcd-settings-title {
     margin-bottom: 0.5rem;
-}
\ No newline at end of file
+}
+
+.tray-icon-wrap {
+    position: relative;
+    top: 0;
+    right: 0;
+}
+
+.tray-icon-image {
+    border-radius: 50%;
+    position: relative;
+    top: 0;
+    right: 0;
+}
+
+.edit-button {
+    visibility: visible;
+    display: block;
+    opacity: 0;
+    position: absolute;
+    top: 4px;
+    left: 4px;
+}
+
+.tray-icon-wrap:hover .tray-icon-image {
+    transition: 0.3s ease;
+    background-color: rgb(0, 0, 0) no-repeat;
+    opacity: 0.25;
+}
+.tray-icon-wrap:hover .edit-button {
+    transition: 0.3s ease;
+    visibility: visible;
+    opacity: 1;
+}
+
+#tray-setting {
+    height: 48px;
+    position: relative;
+}
+#colLeft {
```

Comment:
this css is really cursed

look into css grid/flex layout! but i can also help with the ui if you'd like

Reply:
Thanks for the tip! I changed to a flex layout but this is definitely not my strong suit so feel free to offer some suggestions